alerts/cluster-network-operator/NorthboundStaleAlert

```diff
@@ -0,0 +1,22 @@
+# Northbound Stale Alert
+
+## Meaning
+
+The alert `NorthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
+without succesfully synching any changes to the Northbound Database.
+
+## Impact 
+
+Networking control plane is degraded and Networking Configuration updates applied to the cluster will not be applied.
+Existing workloads should continue to have connectivity, but the OVN-Kubernetes control plane and/or the OVN southbound database
+may not be functional. 
+
+## Diagnosis
+
+Either ovn is overloaded, i.e. the load on the cluster is too large due to a large number of services, network policies, pods, or nodes, or the control plane
+is degraded.
+
+## Mitigation 
+
+In the case that the cluster is overloaded, if the cluster is not on version 4.8, upgrading to that version should help as there have been numerous performance 
```

Comment:
Its not good to mention any versions to update to and from here because this will be out of date very quickly. Please remove this.

You can just say mitigation will depend on what was found in the diagnosis section above.

Reply:
If there is known significant performance issue on particular version of OpenShift with OVN, it should be ok to mention to let the user avoid or start to use OVN after that version?

Reply:
@bpickard22  do you know of any?

Reply:
@martinkennelly I dont know of any specific performance issues, I just know we have improvements on 4.8 and then 4.10 as well